core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -495,9 +475,9 @@ var TopicEditorPage = function() {
   };
 
   this.changeTopicName = async function(newName) {
-    await topicNameField.clear();
-    await topicNameField.sendKeys(newName);
-    await topicNameHeading.click();
+    await action.clear('Topic Name Field', topicNameField);
+    await action.sendKeys('Topic Name Field', topicNameField, newName);
+    await action.click('Topic Name Heading', topicNameHeading);
   };
 
   this.expectTopicNameToBe = async function(name) {
```

Comment:
Can we wait for visibility before calling `.getAttribute`?

Reply:
pleas how can I add awaitFor statement

Reply:
You can use the functions in `waitFor.js`. For examples, look at the `waitFor.visibilityOf` statements in other e2e tests

Reply:
Alright